docs/rancher/index.md

```diff
@@ -0,0 +1,191 @@
+# Rancher Cookbook
```

Comment:
Officially it is now SUSE Rancher

Reply:
Here and in the BTP Cookbooks table of contents please

Reply:
Both updated, I've left the directory as rancher, but updated the mkdocs.yaml to show SUSE Rancher